src/functional/program_processors/codegens/gtfn/gtfn_backend.py

```diff
@@ -22,19 +22,19 @@
 from functional.program_processors.codegens.gtfn.itir_to_gtfn_ir import GTFN_lowering
 
 
-def generate(program: itir.FencilDefinition, **kwargs: Any) -> str:
-    transformed = program
+def generate(program: itir.FencilDefinition, apply_transforms: bool = True, **kwargs: Any) -> str:
```

Comment:
We should introduce a more flexible pass manager, in Cartesian for example we allow to pass a custom optimization pipeline. Maybe the simple option here would be to pass a callable here instead of the boolean which defaults to the current `apply_common_transforms`. But even that feels slightly out of scope for this PR.

Reply:
I'll leave it like that and opened an issue.

Reply:
Okay, that sounds good to me.